src/frame/js/components/action.jsx

```diff
@@ -140,7 +140,8 @@ export class ActionComponent extends Component {
                                        currency={ currency.toUpperCase() }
                                        name={ stripeAccount.appName }
                                        image={ stripeAccount.iconUrl }
-                                       closed={ this.onStripeClose }>
+                                       closed={ this.onStripeClose }
+                                       executionContext={ parent }>
```

Comment:
Parent here I'm guessing is the actual page itself? This is defined in react somewhere?